plugins/SegmentEditor/Model.php

```diff
@@ -129,6 +129,74 @@ public function getSegmentByDefinition($definition)
         return $segment;
     }
 
+    /**
+     * Gets a list of segments that have been deleted in the last week and therefore may have orphaned archives.
+     * @param Date $date Segments deleted on or after this date will be returned.
+     * @return array of segments. The segments are only populated with the fields needed for archive invalidation
+     * (e.g. definition, enable_only_idsite).
+     * @throws \Exception
+     */
+    public function getSegmentsDeletedSince(Date $date)
+    {
+        $dateStr = $date->toString('Y-m-d');
```

Comment:
Better to use `$date->getDatetime()`

Reply:
Actually seeing it seems to be expecting a timestamp so might have to use `getTimestamp()` alternatively